expressions/lookup.js

```diff
@@ -55,7 +55,15 @@ Lookup.prototype.value = function(scope, readOptions){
 		//var propDefined = typeof context === "object" && canReflect.hasKey(context, this.key);
 
 		if (!propDefined) {
-			dev.warn('can-stache/expressions/lookup.js: Unable to find key "' + this.key + '".');
+			var message = 'Unable to find key "' + this.key + '"';
+			if (scope.templateContext.filename) {
+				message = '"' + scope.templateContext.filename + '": ' + message;
+			}
+			var contextName = context && context.constructor.name;
```

Comment:
I believe there's a can-reflect way to get the function name, right @m-mujica ?

Reply:
yup [canReflect.getName](https://github.com/canjs/can-reflect/blob/9c67e35fff3e4e848579d7d17f617751af4b9910/reflections/get-name/get-name.js)... The default behavior would add a `{}` to the name though, I'm not sure if that's an issue here.